metricbeat/module/kubernetes/state_node/state_node.go

```diff
@@ -46,11 +46,12 @@ var mapping = &p.MetricsMapping{
 
 		"kube_node_status_condition": p.LabelMetric("status", "status", p.OpFilterMap(
 			"condition", map[string]string{
-				"Ready":          "ready",
-				"MemoryPressure": "memory_pressure",
-				"DiskPressure":   "disk_pressure",
-				"OutOfDisk":      "out_of_disk",
-				"PIDPressure":    "pid_pressure",
+				"Ready":              "ready",
+				"MemoryPressure":     "memory_pressure",
+				"DiskPressure":       "disk_pressure",
+				"OutOfDisk":          "out_of_disk",
```

Comment:
is this status still available? looking at https://kubernetes.io/docs/concepts/architecture/nodes/#condition - doesn't seem so

could we please adjust it as well?

Reply:
seems this value was deleted in K8s version 1.16 - https://github.com/kubernetes/kubernetes/blob/d89d5ab2680bc74fe4487ad71e514f4e0812d9ce/CHANGELOG/CHANGELOG-1.16.md?plain=1#L1626